tests/unit/sagemaker/test_deserializers.py

```diff
@@ -23,3 +23,27 @@ def test_bytes_deserializer():
     result = deserializer.deserialize(io.BytesIO(b"[1, 2, 3]"), "application/json")
 
     assert result == b"[1, 2, 3]"
+
+
+def test_csv_deserializer_single_element():
+    deserializer = CSVDeserializer()
```

Comment:
it may be more pythonic to define fixture once

```
@pytest.fixture
def csv_deserializer():
    return CSVDeserializer()


def test_1(csv_deserializer):
    ...

def test_n(csv_deserializer):
    ...
```

you can even define scope if desired

Reply:
Fixed